app/views/comments/index.html.erb

```diff
@@ -37,96 +37,113 @@
   <% end %>
 <% end %>
 
-<% if @commentable.class.name == "Article" %>
-  <%= render "shared/payment_pointer", user: @commentable.user %>
-  <div id="comment-article-indicator" data-article-id="<%= @commentable.id %>">
-<% end %>
-<% if @root_comment %>
-  <div class="single-comment-header"></div>
-<% else %>
-  <div class="article-header">
-    <a href="<%= @commentable.path %>" class="header-link">
-      <% if @commentable.respond_to?(:main_image) && @commentable.main_image.present? %>
-        <div class="picture" style="background-image:url(<%= cloud_cover_url(@commentable.main_image) %>)"></div>
-      <% else %>
-        <div class="blank-comment-space"></div>
-      <% end %>
-      <h3 id="comments-header"><%= @commentable.title %></h3>
-      <h4>
-        <%= @commentable.is_a?(PodcastEpisode) ? @commentable.podcast.name : @commentable.user.name %>
-        <span class="published-at"><%= "on #{@commentable.published_at.strftime('%B %d, %Y')}" if @commentable.published_at %></span>
-      </h4>
-    </a>
-    <% if @commentable.processed_html.present? %>
-      <% if @commentable.processed_html.size < 350 %>
-        <div class="body">
-          <%= sanitize_rendered_markdown(@commentable.processed_html) %>
+<div class="crayons-layout crayons-layout--limited-l gap-0">
+  <% if @commentable.class.name == "Article" %>
+    <%= render "shared/payment_pointer", user: @commentable.user %>
+    <span id="comment-article-indicator" data-article-id="<%= @commentable.id %>"></span>
+  <% end %>
+
+  <% if !@root_comment %>
+    <article class="crayons-article crayons-card crayons-card--secondary s:mx-2 m:mx-4 -mb-1 z-0">
+      <header class="crayons-article__header">
+        <% if @commentable.respond_to?(:main_image) && @commentable.main_image.present? %>
```

Comment:
This works for now, but in the long run we may want a better solution leveraging OOP (e.g. `main_image` always returning `nil` for classes that don't have one, so can skip the first check.

Reply:
I think that's a good idea but obviously, not something I'm capable of doing probably.. :C